### PR TITLE
Fix release

### PR DESCRIPTION
**Pull request by crow**
### What do these changes do?
These changes refactor the airship class to airship_flutter to resolve a warning on publish. This allows the plugin to pass the publish dry run that our github actions tool runs before force publishing.

### Why are these changes necessary?
To make it possible to release using the current github actions job

### How did you verify these changes?
Ran both samples, ran and passed publish dry run

#### Verification Screenshots:
image
image

image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+Version 3.0.0 - February 19, 2020
+================================
+- Refactored airship class name to airship_flutter to resolve publish warning
```
Comment (rlepinski) [inline CHANGELOG.md]: Mention what customers will have to do for the update

Comment (marc-scig) [inline CHANGELOG.md]: Do you mean "file name" here? Seems like the class is still Airship.

Comment (marc-scig) [reply CHANGELOG.md]: Did we still need to mention what customers need to do in order to update? I assume we at least want to say something about changing import statements.

Comment (crow) [reply CHANGELOG.md]: it's the dart interface, I guess it's not technically a class

**Pull request merged**